frontend/src/components/core/StreamlitDialog/styled-components.ts

```diff
@@ -46,61 +47,68 @@ export const StyledShortcutLabel = styled.span(({ theme }) => ({
   },
 }))
 
-export const StyledThemeColor = styled.div(({ theme }) => ({
-  display: "flex",
-  alignItems: "center",
-
-  "*": {
-    marginRight: theme.spacing.md,
-  },
+export const StyledBackButton = styled(ChevronLeft)(({ theme }) => ({
+  cursor: "pointer",
+  marginRight: theme.spacing.lg,
 }))
 
-export const StyledThemeCreator = styled.div(({ theme }) => ({
+export const StyledDialogBody = styled.div(({ theme }) => ({
   display: "grid",
-  gridGap: theme.spacing.md,
-  gridTemplateColumns: "1fr",
-}))
-
-export const StyledThemeDesc = styled(Small)(({ theme }) => ({
-  height: "auto",
-  lineHeight: theme.fontSizes.md,
-  marginBottom: theme.spacing.md,
-  display: "block",
-}))
-
-export const StyledThemeCreatorWrapper = styled.div(({ theme }) => ({
-  marginTop: theme.spacing.md,
+  gap: theme.spacing.xl,
+  gridTemplateColumns: "1fr 1fr",
+  marginTop: 0,
```

Comment:
What about just `margin: 0` and `padding: 0`?

Reply:
Ah, yeah the shorthand should work here. Fixed (I think running into the problem fixed in #2938 has conditioned me to not worry about how verbose using the long-form is)